packages/test/end-to-end-tests/src/test/loaderTest.spec.ts

```diff
@@ -157,7 +157,8 @@ describe("Loader.request", () => {
             });
             assert(false, "Loader pause flags doesn't pause container op processing");
         } catch (e) {
-            assert.strictEqual(e.message, `DataStore ${newDataStore.id} does not exist`);
+            const topFrame: string | undefined = e?.stack.split("\n")[1].trimLeft();
```

Comment:
why did this have to change?

Reply:
Assert message changed.  I thought this was a slightly looser coupling.